httpserver/service_node.cpp

```diff
@@ -93,6 +95,33 @@ static std::shared_ptr<request_t> make_push_request(std::string&& data) {
     return make_post_request("/v1/swarms/push", std::move(data));
 }
 
+template <class T>
+static bool verify_message(const T& msg, const char* error_message = nullptr) {
```

Comment:
Why use a template when T is always `message_t`?

Reply:
True, didn't bother checking which struct was used here. Hopefully we soon only handle one struct for messages!